CMakeLists.txt

```diff
@@ -133,6 +133,8 @@ include(external/any)       # download libn::any
 include(external/eigen)     # download eigen3
 include(external/pybind11)  # download pybind11
 include(external/nccl)
+include(external/cares)
+include(external/grpc)
```

Comment:
I vaguely remember that you said you want to try brpc? @typhoonzero

Reply:
Yep. Will add benchmark in next PR to decide which to use. That won't affect current code structure.